src/tennisSimulator/match/ScoreTable.java

```diff
@@ -0,0 +1,112 @@
+package tennisSimulator.match;
+
+import tennisSimulator.config.Configuration;
+import tennisSimulator.player.Player;
+
+public class ScoreTable {
+
+    private final Player firstPlayer;
+    private final Player secondPlayer;
+    private String firstPlayerPoints;
+    private String secondPlayerPoints;
+    private String firstPlayerServingStatus;
+    private String secondPlayerServingStatus;
+    private String matchPhase = " ";
+    private String firstPlayerScoreDisplay;
+    private String secondPlayerScoreDisplay;
+    private String firstPlayerGamesWon;
```

Comment:
Because I didn`t want to display '0' during first call of `scoreTable.show()` during the match, but now I can`t explain my decision :) 

Changed variable type to `int`